### DISCUSSION TITLE
Prevent renovate opening PR to old version of package

### DISCUSSION THREAD

**Discussion by Nirajn2311** [renovatebot/renovate#24499] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I recently added my docker-compose files to a repo for better management and automate their updation. Renovate properly pinned the digests but it opened 2 PRs to update 2 images to an old version which was released around 2 years ago which shouldn't be the case.

The 2 PRs are:
* https://github.com/Nirajn2311/home-server-docker/pull/4
* https://github.com/Nirajn2311/home-server-docker/pull/5

### What you tried so far.

I tried updating the regex for better matching but that dint seem to change anything. For now I've just closed those PRs. Is there a way to make sure when those 2 images get an update renovate will catch the right version and prevent similar PRs for future images which get recognised.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
If images have "bad" versions still on the Docker registry, you can use `allowedVersions` to tell Renovate to only include a certain range, e.g. `<20`

**Reply by Nirajn2311** [reply]
Ok thanks for that. But then wouldn't I have to remove that constraint when the version crosses it. Is there an option to check for updates within a timeframe?

**Reply by rarkins** [reply]
No, and docker tags sometimes don't have timestamps either

**Reply by Nirajn2311** [reply]
Gotcha, thanks. I've gone with your above solution.
